lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -231,6 +231,11 @@
         TargetWidth="144" TargetName="app/src/main/res/mipmap-xxhdpi/icon.png" />
     <ImageFile Name="@(Project.Android.Icons.XXXHDPI:Path || '@//Assets/DefaultIcon.png')"
         TargetWidth="192" TargetName="app/src/main/res/mipmap-xxxhdpi/icon.png" />
+    <CopyFile Name="@(Project.Android.Icons.ANYDPI_v26:Path || '@//Assets/adaptive_icon/mipmap-anydpi-v26/icon.xml')" TargetName="app/src/main/res/mipmap-anydpi-v26/icon.xml" />
```

Comment:
I am worried that this would make `icon.xml` unintentionally override user-specified icons in all existing apps, which would be a breaking change. Effectively the change would force all existing apps to provide a new `icon.xml` to restore their original icon.

So after a second though I am not sure if this is the right way forward... Perhaps use `<CopyFile Condition="USE_ADAPTIVE_ICON" ... />`, only copying the new files when `-DUSE_ADAPTIVE_ICON` is specified?

Reply:
Ok, almost there...